exla/lib/exla.ex

```diff
@@ -360,11 +360,18 @@ defmodule EXLA do
   Takes in a function, the argument templates and the compilation
   options and returns the textual representation of the MLIR module.
 
+  ## Options
+
+    * `:nested_defn_compilation` - a boolean that indicates whether
```

Comment:
Could we check `Nx.Defn.Compiler.defn?()` instead?

Reply:
I tried, but that only gets set inside the runtime fun call, and that's after what I need

Reply:
What about `Nx.Defn.Compiler.current()`?